### PR TITLE
Add documentation for Garmin Connect

### PR DESCRIPTION
**Pull request by cyberjunky**
**Description:**

**Pull request in home-assistant (if applicable):** home-assistant/home-assistant#30792
## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_integrations/garmin_connect.markdown [hunk 1]
```diff
@@ -0,0 +1,126 @@
+---
+title: Garmin Connect
+description: Instructions on how to configure the Garmin Connect integration for Home Assistant.
+logo: garmin_connect.png
+ha_category:
+  - Health
+ha_iot_class: Cloud Polling
+ha_release: 0.19
```
Comment (frenck) [inline source/_integrations/garmin_connect.markdown]: This seems unlikely 😉

File: source/_integrations/garmin_connect.markdown [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+ha_codeowners:
+  - '@cyberjunky'
+---
+
+The Garmin Connect sensor allows you to expose data from [Garmin Connect](https://connect.garmin.com) to Home Assistant.
+
```
Comment (frenck) [inline source/_integrations/garmin_connect.markdown:14]: Add a `# Configuration` header

File: source/_integrations/garmin_connect.markdown [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+Enable the sensor by adding the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+sensor:
+  - platform: garmin_connect
+    email: GARMIN_CONNECT_EMAIL
+    password: GARMIN_CONNECT_PASSWORD
+    resources:
```
Comment (frenck) [inline source/_integrations/garmin_connect.markdown]: We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.

File: source/_integrations/garmin_connect.markdown [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+      - "maxHeartRate"
+      - "maxStressLevel"
+```
+
+Restart Home Assistant once this is complete.
+
+Please be aware that Garmin Connect has very low rate limits, max. once every 10 minutes.
+
```
Comment (frenck) [inline source/_integrations/garmin_connect.markdown:22]:
```suggestion
```

Unneeded additional empty line.

**Review by frenck — CHANGES_REQUESTED** [review]
Thanks for the PR @cyberjunky!

I've reviewed it and left some comments. Could you please take a look? Thanks! 👍

File: source/_integrations/garmin_connect.markdown (hunk 3 above)
Comment (cyberjunky) [reply source/_integrations/garmin_connect.markdown]: Understood, and fixed.

File: source/_integrations/garmin_connect.markdown (hunk 2 above)
Comment (cyberjunky) [reply source/_integrations/garmin_connect.markdown:14]: Added

File: source/_integrations/garmin_connect.markdown (hunk 1 above)
Comment (cyberjunky) [reply source/_integrations/garmin_connect.markdown]: Copy and paste error ;)

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.
⚠️ The `ha_release` might need an update before merging this one in.

**Comment by cyberjunky** [comment]
Thanks, I'm still struggling with the correct/wanted config name for the resources.
I'm now using the home-assistant's native in the code:
```
CONF_MONITORED_CONDITIONS
```
But FitBit uses a local declared:
```
CONF_MONITORED_RESOURCES = "monitored_resources"
```
And I believe there is also a:
```
CONF_RESOURCES
```
Any suggestion/preferences?

**Comment by frenck** [comment]
That is an implementation discussion, which is ok to have, but please do so on the codebase PR.

**Review by frenck — DISMISSED** [review]

**Pull request merged**